notascript/src/CreateDocument.js

```diff
@@ -2,8 +2,10 @@ import React from 'react';
 import Form from './Form';
 import PropType from 'prop-types';
 
-export default function CreateDocument({ onFormSubmit, props }) {
-	return <Form onFormSubmit={onFormSubmit} {...props} />;
+export default function CreateDocument({ onFormSubmit, props, domainList }) {
```

Comment:
and here as well, if you move history.replace() to the Form component, you need the props if you leave it in the App you don't need to hand over props to the Form component